src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_help.py

```diff
@@ -284,6 +284,18 @@
     type: command
     short-summary: Update a server.
     """
+helps['sql server conn-policy'] = """
+    type: group
+    short-summary: Manage a server's connection policy.
+    """
+helps['sql server conn-policy show'] = """
+    type: command
+    short-summary: Gets a server's secure connection policy.
+    """
+helps['sql server conn-policy update'] = """
```

Comment:
Is this update patch-like behavior?  I don't see a create command so I'm assuming it's a create or update kind of update?  If so that's fine, but if it is more like a PUT then the command should be called set.

Reply:
The resource always exists, and the --connection-type property can be modified. Currently --connection-type is the only mutable property so it is required, but in theory new properties might be added in the future and it would be nice if it did patch-like behavior (using GET and then PUT). So is `update` good?